screens/UserScreens/UserDashboard.jsx

```diff
@@ -1,15 +1,50 @@
 import { Modal, View, Text, ScrollView } from "react-native";
-import CreateAndJoinButtons from "../../components/UserDashboard/CreateAndJoin";
-import ProductionButton from "../../components/UserDashboard/ProductionButton";
+import { useState, useEffect } from "react";
+import { onValue, get, ref, child, set } from "firebase/database";
 import { FontAwesomeIcon } from "@fortawesome/react-native-fontawesome";
 import { faClapperboard, faMasksTheater } from "@fortawesome/free-solid-svg-icons";
-import { mockProductions } from "../../MockData/MockProductions";
-import { useState } from "react";
+import { firebase_auth, firebase_db } from "../../firebase.config.js";
+import CreateAndJoinButtons from "../../components/UserDashboard/CreateAndJoin";
+import ProductionButton from "../../components/UserDashboard/ProductionButton";
 import JoinProductionModal from "../../components/ProductionModals/JoinProductionModal";
 import CreateProductionModal from "../../components/ProductionModals/CreateProductionModal";
 
+//TODO:
```

Comment:
Remove TODO Unless still Relevant (First bit defo isn't lol)

Reply:
yup realised that just after making the pr